tfexec/show.go

```diff
@@ -8,15 +8,44 @@ import (
 	tfjson "github.com/hashicorp/terraform-json"
 )
 
+type showConfig struct {
+	reattachInfo ReattachInfo
+}
+
+var defaultShowOptions = showConfig{}
+
+type ShowOption interface {
+	configureShow(*showConfig)
+}
+
+func (opt *ReattachOption) configureShow(conf *showConfig) {
```

Comment:
Does `show` uses provider binaries? Not sure, I assumed it only used backend.

Reply:
Apparently it does something like that, because I got an error when I didn't supply the reattach env var...

Reply:
👍